mystery_word.py

```diff
@@ -0,0 +1,177 @@
+import random
+import string
+
+
+def open_file(file_input):
+    words = [line.strip() for line in open(file_input)]
+    return words
+
+
+def get_random_word():
+    words = open_file('/usr/share/dict/words')
+    random_word = random.choice(words)
+    return random_word
+
+
+def word_length(word):
+    length = len(word)
+    print("The computer's word is {} letters".format(length))
+    return length
+
+
+def valid_user_input(user_guess):
+    if len(user_guess) > 1:
+        return False
+    elif user_guess in string.punctuation:
+        return False
+    else:
+        return True
+
+
+def guess_in_word(guess, random_word):
+    lowercase_guess = guess.lower()
+    for character in random_word:
+        if lowercase_guess == character:
+            return True
+            user_guesses.extend(lowercase_guess)
+            break
+    return False
+
+
+def hide_letters(user_guesses, random_word):
+    word_display = []
+    for character in random_word:
+        if character in user_guesses:
+            word_display.extend(character)
+        else:
+            word_display.extend("_")
+    return word_display
+
+
+def check_guesses_remaining(guesses_remaining):
+    if guesses_remaining > 0:
+        return True
+    else:
+        return False
```

Comment:
This could be simplified to:

```py
def check_guesses_remaining(guesses_remaining):
    return guesses_remaining > 0
```

This is a common thing people do when they're starting. Remember that the clause in the if statement evaluates to True or False, so you can just return it.